CONTRIBUTING.md

```diff
@@ -354,9 +336,8 @@ For context, check out the discussion in issue [#242][242].
 [authy]: https://authy.com/
 [duo]: https://duo.com/
 [googleauthenticator]: https://github.com/google/google-authenticator
-[jekyll]: https://jekyllrb.com/
-[bundler]: https://bundler.io/
-[gemfile]: /Gemfile
+[README]: https://github.com/2factorauth/twofactorauth#installing-dependencies-hammer_and_wrench
+[tests]: https://github.com/2factorauth/twofactorauth/blob/master/.github/workflows/repository.yml#L23
```

Comment:
Is is really useful to link to the block by giving the line number? I bet that nobody will care to update this.